tests/test_sqlite_vsts_ci.py

```diff
@@ -0,0 +1,31 @@
+# This is an example test settings file for use with the Django test suite.
+#
+# The 'sqlite3' backend requires only the ENGINE setting (an in-
+# memory database will be used). All other backends will require a
+# NAME and potentially authentication information. See the
+# following section in the docs for more information:
+#
+# https://docs.djangoproject.com/en/dev/internals/contributing/writing-code/unit-tests/
+#
+# The different databases that Django supports behave differently in certain
+# situations, so it is recommended to run the test suite against as many
+# database backends as possible.  You may want to create a separate settings
+# file for each of the backends you test against.
+
+DATABASES = {
+    'default': {
+        'ENGINE': 'django.db.backends.sqlite3',
+    },
+    'other': {
+        'ENGINE': 'django.db.backends.sqlite3',
+    }
+}
+
+SECRET_KEY = "django_tests_secret_key"
+
+# Use a fast hasher to speed up tests.
+PASSWORD_HASHERS = [
+    'django.contrib.auth.hashers.MD5PasswordHasher',
+]
+
+TEST_RUNNER = 'xmlrunner.extra.djangotestrunner.XMLTestRunner'
```

Comment:
Would it be worth changing this to the code below:

```python
from .test_sqlite import *
TEST_RUNNER = 'xmlrunner.extra.djangotestrunner.XMLTestRunner'
````
This would reduce duplication, until when/if runtests.py supports passing `TEST_RUNNER` via a flag or environment variable?

Reply:
Great idea - will do!